R/separate-rows.R

```diff
@@ -31,7 +31,7 @@ separate_rows.data.frame <- function(data,
                                      convert = FALSE) {
   vars <- tidyselect::eval_select(expr(c(...)), data)
 
-  out <- purrr::modify_at(data, vars, stringi::stri_split_regex, pattern = sep)
+  out <- purrr::modify_at(data, vars, strsplit, split = sep)
```

Comment:
I _think_ you should probably set `perl = TRUE` here (and in all other functions where the user supplies the match) since that'll make the regexp engine as similar as possible to stringi.